woof-code/rootfs-petbuilds/geany-gtk2/petbuild

```diff
@@ -1,13 +1,13 @@
 download() {
-    [ -f geany-1.37.1.tar.bz2 ] || wget -t 3 -T 60 https://download.geany.org/geany-1.37.1.tar.bz2
+    [ -f geany-2.0.tar.bz2 ] || wget -t 3 -T 60 https://download.geany.org/geany-2.0.tar.bz2
 }
 
 build() {
-    tar -xjf geany-1.37.1.tar.bz2
-    cd geany-1.37.1
+    tar -xjf geany-2.0.tar.bz2
+    cd geany-2.0
     patch -p1 < ../defaults.patch
-    ./configure --prefix=/usr --disable-plugins --disable-vte --enable-gtk2
+    ./configure --prefix=/usr --enable-plugins --enable-vte --enable-gtk2
```

Comment:
@lakshayrohila Are you sure that Geany 2.0 supports GTK+ 2? AFAIK --enable-gtk2 isn't supported and this petbuild actually produces a GTK+ 3 build. Originally, I copied `geany` to `geany-gtk2` and made some build configurations use `geany-gtk2` to avoid breaking Puppy releases based on old Ubuntu or Slackware versions, too old for latest Geany.

Reply:
Well, yes, I didn't test that! I ran it and it worked well, so I thought it might be supported. Which older puppy can I build to test if this works. For Jammy64, I had changed from using geany to geany-gtk2 in __0config.sh

Reply:
The main question is - is this update even required and brings something new?

Reply:
Look at the Geany changelog and decide if you want to update or not 🤷🏿

I'd recommend leaving geany-gtk2 alone, because it's used by build configurations that can't build the latest Geany (stuff like slackware/14.2) where GTK+ 3 is too old.

Reply:
@dimkr, I think there are some things which can be useful for some Puppy users in this update. The only problem is those extra 11 MB it brings with it.

Reply:
So update `geany` and leave `geany-gtk2` alone